Gcpe.Hub.API/Controllers/MessagesController.cs

```diff
@@ -25,6 +25,36 @@ public MessagesController(HubDbContext dbContext, ILogger<MessagesController> lo
             this.mapper = mapper;
         }
 
+        // Bumps the sort order of all published messages from firstSortOrder to lastSortOrder
+        // A negative lastSortOrder omits it from the query
+        public static int BumpSortOrders(HubDbContext dbContext, Boolean positiveDirection, int firstSortOrder, int lastSortOrder)
+        {
+            IQueryable<Message> messages;
+            if (lastSortOrder >= 0)
+            {
+                messages = dbContext.Message.Where(m => m.IsPublished && m.IsActive && m.SortOrder >= firstSortOrder && m.SortOrder <= lastSortOrder);
+            }
+            else
+            {
+                messages = dbContext.Message.Where(m => m.IsPublished && m.IsActive && m.SortOrder >= firstSortOrder);

```

Comment:
Make this the default messages querables and add m.SortOrder <= lastSortOrder as a second where clause